src/software/ai/hl/stp/play/ball_placement/ball_placement_play_fsm.cpp

```diff
@@ -137,171 +207,194 @@ void BallPlacementPlayFSM::retreat(const Update &event)
                 closer_goal = world_ptr->field().enemyGoalCenter();
             }
             retreat_direction = (closer_goal - ball_pos).normalize();
-            retreat_position  = ball_pos + retreat_direction * (RETREAT_DISTANCE_METERS +
+            retreat_position  = ball_pos + retreat_direction * (RETREAT_DISTANCE_M +
```

Comment:
`retreat_position` calculation isn't consistent with how it's computed above. Maybe a helper? `RETREAT_DISTANCE_M` should also include the `ROBOT_MAX_RADIUS` accounting.